docs/operator-docs/cmr-operations.md

```diff
@@ -0,0 +1,63 @@
+---
+id: cmr-operations
+title: CMR Operations
+hide_title: true
+---
+
+# CMR Operations
```

Comment:
If this is going to be an operator doc, it needs to show how to do this doing the dashboard

Reply:
The tricky part there is I can't provide screenshots of how to do everything with the dashboard until the dashboard is updated.